src/main/scala/mesosphere/marathon/raml/QueueInfoConversion.scala

```diff
@@ -62,8 +63,8 @@ trait QueueInfoConversion extends DefaultConversions with OfferConversion {
       }
 
       info.runSpec match {
-        case app: AppDefinition => queueItem(QueueApp(_, _, _, _, _, Raml.toRaml(app)))
-        case pod: PodDefinition => queueItem(QueuePod(_, _, _, _, _, Raml.toRaml(pod)))
+        case app: AppDefinition => queueItem(QueueApp(_, _, _, _, _, _, Raml.toRaml(app)))
+        case pod: PodDefinition => queueItem(QueuePod(_, _, _, _, _, _, Raml.toRaml(pod)))
```

Comment:
I understand what it does, but... It's pretty ugly. Is there any cool way to improve this?

Reply:
LOL, I agree :) Seems like a job for the factory pattern